vtex-sites/nextjs.store
# fix: Display `SearchDropdown` correctly

## What's the purpose of this pull request?

- Attempts to fix `SearchDropdown` displaying below navlinks
- Follow-up https://github.com/vtex-sites/nextjs.store/pull/191#pullrequestreview-1061668003

## How does it work?

Changes back `z-index: var(--fs-z-index-below)`

|Current|Fixed|
|-|-|
|<img width="1154" alt="image" src="https://user-images.githubusercontent.com/3356699/185451624-b40bd241-147b-484e-a5fe-dae9ad8d2ce6.png">|<img width="1423" alt="image" src="https://user-images.githubusercontent.com/3356699/185452126-531a5962-849d-4434-bbeb-8a31e241545b.png">|


## How to test it?

- Check on the [preview](https://preview-212--nextjs.preview.vtex.app) if the search dropdown is being displayed correctly.
- Also verify if [navlinks](https://nextjs-store-storybook-git-fix-navlink-z-index-faststore.vercel.app/?path=/docs/organisms-navbar-navlinks--overview-default) is appearing on the stories.
<img width="1192" alt="image" src="https://user-images.githubusercontent.com/3356699/185616433-7032654c-272e-4dac-af1c-9ca5bb27a5be.png">

 
## Checklist

<em>You may erase this after checking them all :wink:</em>

**PR Title and Commit Messages**
  - Available prefixes: `feat`, `fix`, `chore`, `docs`, `style`, `refactor`, `perf`, and `test`

**PR Description**
- [x] Added a label according to the PR goal - `Breaking change`, `Features`, `Bug fixes`, `Chore`, `Documentation`, `Style changes`, `Refactoring`, `Performance`, and `Test`
- [x] Added the component, hook, or path name in-between backticks (\`\`) - *if applicable, e.g., `ComponentName` component, `useWindowDimensions` hook*

**Dependencies**

**Documentation**
